Applications/SimpleUI/static/simpleui.js

```diff
@@ -0,0 +1,103 @@
+
+        // Build URL for Telemetry db query, then retrieve data
+        var url = "http://localhost:8001/?limit=1&callsign=" +
+            callsign + // jshint ignore:line
+            "&nodeid=" + 
+            nodeid; // jshint ignore:line
+            $.getJSON(url,function(data){
+                try{
+                    // Convert EPOCH from telemetry into date object
+                    var epoch = data[0].EPOCH;
+                    var d = new Date(0);
+                    d.setUTCSeconds(epoch);
+                    $(".EPOCH").text(d);
+
+                    // Populate the GPS section
+                    $(".CALLSIGN").text(data[0].SOURCECALLSIGN);
+                    $(".NODEID").text(data[0].SOURCEID);
+                    $(".GPSLATITUDE").text(data[0].GPSLATITUDE);
+                    $(".GPSLATITUDEDIR").text(data[0].GPSLATITUDEDIR);
+                    $(".GPSLONGITUDE").text(data[0].GPSLONGITUDE);
+                    $(".GPSLONGITUDEDIR").text(data[0].GPSLONGITUDEDIR);
+                    $(".ALTITUDE").text(data[0].GPSALTITUDE);
+                    $(".SPEED").text(Math.round(data[0].GPSSPEED*0.514444*100)/100);
+                    $(".GPSFIX").text(data[0].GPSFIX);
+
+                    // Populate the ADC section
+                    $(".BOARDTEMP").text(data[0].BOARDTEMP);
+                    $(".ADC0").text(Math.round(data[0].ADC0*1000*(2.41/4096))/1000);
+                    $(".ADC1").text(Math.round(data[0].ADC1*1000*(2.41/4096))/1000);
+                    $(".ADC2").text(Math.round(data[0].ADC2*1000*(2.41/4096))/1000);
+                    $(".ADC3").text(Math.round(data[0].ADC3*1000*(2.41/4096))/1000);
+                    $(".ADC4").text(Math.round(data[0].ADC4*1000*(2.41/4096))/1000);
+                    $(".ADC5").text(Math.round(data[0].ADC5*1000*(2.41/4096))/1000);
+
+                    // Populate the High Altitude Balloon section
+                    $(".HABTIMER").text(data[0].HABTIMER);
+                    $(".HABTIMERSTATE").text(data[0].HABTIMERSTATE);
+                    $(".HABTRIGGERTIME").text(data[0].HABTRIGGERTIME);
+                    $(".HABCUTDOWNSTATE").text(data[0].HABCUTDOWNSTATE);
+
+                    // Convert GPIO data into an array of GPIO values
+                    var gpio = [];
+                    gpio = ("000000000" + data[0].GPIOSTATE.toString(2)).substr(-8).split("");
+
+                    // Convert IOSTATE data into array of binary values
+                    var iostate = [];
+                    iostate = ("000000000" + data[0].IOSTATE.toString(2)).substr(-8).split("");
+                    iostate.reverse();
+
+                    // Convert RFSTATE data to an array of values
+                    var rfstate = [];
+                    rfstate = ("000000000" + data[0].RFSTATE.toString(2)).substr(-8).split("");
+                    rfstate.reverse();
+
+                    // Populate GPIO data in IO section
+                    $(".GPIO0").text(gpio[0]);
```

Comment:
(With updated firmware per my firmware issue ticket)

When I command GPIO 7 I see GPIO 0 turn on:
![image](https://cloud.githubusercontent.com/assets/339144/25201025/55a8040e-2505-11e7-86ba-deaabbfe9810.png)

Reviewing this in code composer shows that I properly bit shifted GPIO 7 into the MSB location that matches the excel document:

![image](https://cloud.githubusercontent.com/assets/339144/25201070/7e82305c-2505-11e7-900e-b65062c75780.png)

![image](https://cloud.githubusercontent.com/assets/339144/25201098/961cfaee-2505-11e7-8ad0-6d613a530a67.png)

@kb1lqc please review you're code here to make sure that you are properly allocating the bitfield.